src/components/MissionHubButton.stories.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+import MissionHubButton from './MissionHubButton'
+
+export default {
+  title: 'Components/MissionHubButton',
+  component: MissionHubButton,
+  parameters: {
+    backgrounds: {
```

Comment:
Are these parameters used?

Reply:
yeah, just allows you to change the background of the canvas